requirements/system/user-client.tex

```diff
@@ -1,9 +1,49 @@
 \subsection{User Client}
-\begin{enumerate}
- \item UCs must be accessible through a CLI.
- \item UCs must provide to the user the same functionality as if the user was working directly on a work machine.
- \item UCs must allow the user to specify an operating system on which the job should be run.
- \item UCs must allow the user to specify minimum hardware requirements for a job: RAM, CPU thread count, GPU memory/model/speed.
- \item UCs must allow the user to specify a priority for a job.
-\end{enumerate}
+
+
+  \subsubsection{Functional Requirements} 
+  \begin{itemize}
+    \item [FRUC01] Logging in to the central server
+    \item [FRUC02] Logging out from the central server
+    \item [FRUC05] Queueing new jobs
+    \item [FRUC06] Querying the state of jobs
+    \item [FRUC07] Canceling jobs
+    \item [FRUC08] Receiving the output
+    \item [FRUC09] Setting job priorities 
+    \item [FRUC10] Checking the specifications of available containers
+    \item [FRUC11] Choosing the parameters of the container(preferred OS/environment)
+    \item [FRUC12] Choosing the running mode: "blocking mode"/ default mode
+    \item [FRUC13] Choosing the minimal hardware requirements for a job 
+    \item [FRUC14] Checking the state of the entire server
+  \end{itemize}
+
+  \subsubsection{Non Functional Requirements} 
+  \begin{itemize}
+    \item [NFRUC01] Responses from the server must take less 10ms
```

Comment:
I am not sure we can guarantee anything regarding response times because we do not control the network setup and connections.

Reply:
This has nothing to do with UC having control over the speed or anything. It's just a non functional requirement like we saw in the SWT (otherwise the UC won't have enough NFR)

Reply:
And i m not going to add details until you approve these requirements otherwise it's gonna be  a waste of time

Reply:
We can't just requirements because "the UC won't have enough NFR", we need to add requirements which make sense. If it turns out we have no NFR for UC, then I'd say we can simply skip the whole section.

Reply:
If you find any NFR pls tell me. I can't find any